comm/src/main/scala/coop/rchain/catscontrib/monadOps.scala

```diff
@@ -1,9 +1,16 @@
 package coop.rchain.catscontrib
 
+import ski._
 import cats._, cats.data._, cats.implicits._
 
 final class MonadOps[F[_], A](val self: F[A])(implicit val F: Monad[F]) {
   def liftM[G[_[_], _]](implicit G: MonadTrans[G]): G[F, A] = G.liftM(self)
+  def forever: F[A]                                         = MonadOps.forever(self)
+}
+
+object MonadOps {
+  def forever[F[_]: Monad, A](fa: F[A]): F[A] =
+    fa >>= κ(forever(fa))
```

Comment:
I think using itty-bitty kappa for K is super neat-looking. Is it common? Is it as fraught with problems as I suspect it is?

Reply:
I believe is neat-looking, I don't know what kind of problems you are anticipating. Doing FP requires me to use `constant` and `identity` function ALOT, thus having nice symbol for it, pays off in terms of readability (you make your brain adjust to a handful set of symbols most often used) // CC @nashef